README.md

```diff
@@ -95,9 +95,32 @@ Backend fields:
 - `poll` is the time in seconds between polling for changes.
 - `onChange` is the executable (and its arguments) that is called when there is a change in the list of IPs and ports for this backend.
 
+Service Discovery Backends:
+
+Must supply only one of the following
+
+- `consul` configures discovery via [Hashicorp Consul](https://www.consul.io/). Expects `hostname:port` string:
+
+    ```
+    "consul": "consul:8500"
+    ```
+
+- `etcd` configures discovery via [CoreOS etcd](https://coreos.com/etcd/). Expects a config object:
+
+    ```
+    "etcd": {
+        "endpoints": [
+            "http://etcd:4001"
+        ],
+        "prefix": "/containerbuddy"
+    }
+    ```
+
+    - `endpoints` is the list of etcd nodes in your cluster
+    - `prefix` is the path that will be prefixed to all service discovery keys.
```

Comment:
`prefix` should be marked as optional and defaulting to `/containerbuddy`, if I'm reading the intent of the code right.